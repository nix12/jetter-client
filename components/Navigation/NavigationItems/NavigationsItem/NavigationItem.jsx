import React from 'react';
import Link from 'next/link';

import MenuItem from '@material-ui/core/MenuItem';

const navigationItem = React.forwardRef((props, ref) => {
  const { children, link, click, linkAs } = props;

  return (
    <MenuItem ref={ ref } onClick={ click }>
      <Link href={ link } as={ linkAs }>
        <a>{ children }</a>
      </Link>
    </MenuItem>
  )
})

export default navigationItem;
