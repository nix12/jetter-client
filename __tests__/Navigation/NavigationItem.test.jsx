import React from 'react';
import { shallow } from 'enzyme';
import NavigationItem from '../../components/Navigation/NavigationItems/NavigationItem/NavigationItem';

describe('navigationItem', () => {
  it('should render navigationItem component', () => {
    expect(shallow(<NavigationItem />)).toMatchSnapshot();
  });
});
