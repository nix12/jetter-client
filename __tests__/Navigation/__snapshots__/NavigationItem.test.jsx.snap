// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navigationItem should render navigationItem component 1`] = `ShallowWrapper {}`;
